### ISSUE TITLE
Decode HTML entities when outputting plain text

### ISSUE THREAD

**Issue by larsgw** [larsgw/citation.js#74]
Example output:

```
$ echo Q30000000 | citation-js -f string -s citation-apa
[set] Unknown property: P1922
[set] Unknown property: P407
Could not use "nc", falling back to slower node.js method for sync requests.

  Miccadei, S., Leo, R. D., Zammarchi, E., Natali, P. G., &#38; Civitareale, D. (2002). The Synergistic Activity of Thyroid Transcription Factor 1 and Pax 8 Relies on the Promoter/Enhancer Interplay. Molecular Endocrinology, 16(4), 837–846. https://doi.org/10.1210/MEND.16.4.0808
```

**Comment by larsgw** [comment]
Should be fixed with #81, using text mode, which shouldn't escape characters like that.

**Comment by larsgw** [comment]
Fixed with #81.
